fs_src/script.js

```diff
@@ -849,11 +849,16 @@ function onLoad() {
       // check for update only once when loading the page (not each time in getInfo)
       if (lastInfo.wifi_rssi !== 0) {
         var last_update_check = parseInt(getCookie("last_update_check"));
+        var cached_curVersion = getCookie("cached_curVersion");
```

Comment:
you don't need to use a cookie to store this, a global variable would be sufficient.

Reply:
sorry but do i do that ?

Reply:
i tried

```
var cached_curVersion = null;
async function checkUpdate() {
  var model = lastInfo.model;
.....
      }
      console.log("Version:", latestVersion, "URL:", updateURL);
      var cached_curVersion = curVersion
      if (!latestVersion || !updateURL) {
        console.log("Update section not found:", model, curVersion, cfg);
```

but after a fresh and auto refresh starts i get this.
![Screenshot 2021-02-14 at 00 18 56](https://user-images.githubusercontent.com/77936/107864989-3c237200-6e5a-11eb-9a47-59518c348e21.jpg)

and update banner remains